client/app/components/Input/InputCheckboxGroup.jsx

```diff
@@ -6,7 +6,7 @@ import type { Checkbox } from './utils';
 export type Props = {
   checkboxes: Checkbox[],
   required?: boolean, // At least one checkbox must be checked
-  hasError?: Function,
+  hasError?: (errorPresent: boolean) => void,
```

Comment:
Specified the type here to reflect the fix in the mock prop passed to the component in the test.